include/smack/SmackRep.h

```diff
@@ -184,8 +186,10 @@ class SmackRep {
   virtual vector<string> getModifies();
   unsigned numElements(const llvm::Constant* v);
   void addInit(unsigned region, const llvm::Value* addr, const llvm::Constant* val);
+  void addInitFunc(const Stmt* s);
```

Comment:
I would change the signature of this so that it takes "llvm::Function*" as its parameter. I think that would be a slightly better encapsulation.